source/manual/create-a-gpg-key.html.md

```diff
@@ -61,10 +61,10 @@ The key ID is `90E65803`, and the fingerprint is `37CC 021A C5C2
 
 ### Upload your GPG key to a keyserver
 
-Before doing this, make sure that your default keyserver is `http://keyserver.ubuntu.com/`. You can do this by changing the default keyserver in `~/.gnupg/gpg.conf`:
+Before doing this, make sure that your default keyserver is `https://keyserver.ubuntu.com/`. You can do this by changing the default keyserver in `~/.gnupg/gpg.conf`:
 
 ```
-keyserver http://keyserver.ubuntu.com/
+keyserver https://keyserver.ubuntu.com/
```

Comment:
Good change 👍 as you've said elsewhere in the PR, this server is problematic, and we've had more luck recently with keys.openpgp.org. This is what I have in my `~/.gnupg/dirmngr.conf`, which works well for me:

```
keyserver hkps://keys.openpgp.org
```

I agree it is probably worth changing our recommended server to this keys.openpgp.org one, in a new commit that justifies the reasoning.